pkg/clustermanager/controller.go

```diff
@@ -167,7 +167,6 @@ func (c *controller) processNextItem() bool {
 	defer c.queue.Done(i)
 
 	if err := c.process(item); err != nil {
-		runtime.HandleError(err)
```

Comment:
I would also log the error, at a numerically high `-v` level too.  It is not necessarily an error, so I would log it with `.V(something).InfoS`, but it is potentially an error, this is why I would log it.